app/workers/form1095/get_new1095_bs.rb

```diff
@@ -0,0 +1,176 @@
+require 'sidekiq-scheduler'
+
+module Form1095
+  class GetNew1095Bs
+    include Sidekiq::Worker
+
+
+    def bucket
+      @bucket ||= Aws::S3::Resource.new(
+        region: Settings.form1095_b.s3.region,
+        access_key_id: Settings.form1095_b.s3.aws_access_key_id,
+        secret_access_key: Settings.form1095_b.s3.aws_secret_access_key
+      ).bucket(Settings.form1095_b.s3.bucket)
+      # @bucket ||= Aws::S3::Resource.new(
+      #   region: 'us-gov-west-1', 
+      #   access_key_id: "ASIAQD72FDTF63A3VAUG",
+      #   secret_access_key: "cQ7SC6SvAFULPxMYhp8xjzK9gFagoQFPdNrto9EX",
```

Comment:
@stephte Please disable this key in AWS immediately. This is a security issue. Please be cautious not to check in secrets.